etna/lib/etna/spec/vcr.rb

```diff
@@ -30,6 +48,10 @@ def request_1.path
       end
     end
 
+    c.register_request_matcher :try_uri do |request_1, request_2|
+      clean_query(request_1.uri) == clean_query(request_2.uri)
```

Comment:
For VCR matching, however, it is important to clear non deterministic url components in all cases.